pkg/planner/core/find_best_task.go

```diff
@@ -1423,15 +1423,6 @@ func (ds *DataSource) FindBestTask(prop *property.PhysicalProperty, planCounter
 				}
 			}
 			if canConvertPointGet {
-				// If the schema contains ExtraPidColID, do not convert to point get.
-				// Because the point get executor can not handle the extra partition ID column now.
-				// I.e. Global Index is used
-				for _, col := range ds.schema.Columns {
-					if col.ID == model.ExtraPidColID {
-						canConvertPointGet = false
-						break
-					}
-				}
```

Comment:
I think the code is useless now. We have test cases in `tests/integrationtest/t/globalindex/point_get.test`, and it could generate the `point_get` and `batch_point_get` plan with this code.